jest.config.unit.test.js

```diff
@@ -50,6 +50,8 @@ module.exports = {
         "!src/DomainServices/UserActivity/UserActivityTrackingService.ts",
         "!src/DataAccess/applyMiddleware.ts",
         "!src/DomainServices/Pressroom/PressroomService.ts",
+        "!src/DomainServices/Document/DocumentService.ts",
+        "!src/DomainServices/Snapshot/SnapshotService.ts",
```

Comment:
I removed these services from our unit testing, these 2 services only access the DB. Is that okay? @mnatsheh

Reply:
I'll resolve this, please un-resolve if that's not okay.